theme/views/partials/header.nunj

```diff
@@ -1,5 +1,5 @@
 <div class="Header">
-  <button class="Header-button Header-navToggle" data-action="toggle-sidebar">
+  <button class="Header-button Header-navToggle" data-action="toggle-sidebar" aria-label="menu">
```

Comment:
**Question**: if this is a toggle for the menu - did you want to change this to a `aria-role`? or change what the label since this is the toggle and not the menu itself?

Reply:
Thanks for the comment, it actually got me thinking:
This button shouldn't have `aria-label`/`role` of `menu`, since it's not directly a part of the menu, it just has an action that toggles the menu, which is not part of the same exact tree branch, if you get what I mean.

Now I'm thinking of just giving it `aria-label="toggle menu"` because it's just a toggle to close/open the menu, and that should be clear for screen readers. I do want to also place `aria-label` on both the close toggle and open toggle, however they are `div`s (see here: https://github.com/mozilla/protocol/blob/main/theme/views/partials/header.nunj#L3), and according to this [accessibility site](https://www.davidmacd.com/blog/does-aria-label-override-static-text.html), adding `aria-label` on a `div` will **only** work if it also comes with a `role` attribute. However, I'm not sure which role I should give the `div` in this case if its just holding an icon? Maybe it should be `role="img"`, but I don't think that would be correct in this case, happy to hear what other devs have to say about this.

Reply:
The most semantic solution here would just be `<button>Toggle menu</button>` instead of fiddling around with aria attributes, however that would require a larger refactor of the button, which may not be possible given ithe current markup is something Fractal could rely on.

So, instead I think using `aria-label="Toggle menu"` makes sense to me @reemhamz. However, I'm not sure you really need to go further with adding additional attributes to the `<div>`'s, as these are child elements of the `<button>` which has the required semantics?

Adding an `aria-controls` attribute to the menu icon could be an additional improvement to help screen readers understand what the button does when clicked.

Reply:
Thanks for your insight Alex!
I wanted to add additional aria attributes to the `div`'s because one was to close the menu, and the other to open it, and each option had a respective icon to go with it. 

I don't think, depending on the hierarchy of how the button is placed several layers prior to the actual `nav` element, that it would make a difference here to screenreaders, especially considering that even if the sidemenu isn't showing, the screenreader is still going through the options. 
I suppose this is something from Fractal's end, and we'd have to do more refactoring if we want it to be optimal.
For now, I'll stick to adding the `aria-label="Toggle menu"` attribute on the button element.